src/model_signing/_signing/signing.py

```diff
@@ -211,6 +212,13 @@ class Payload:
       ],
       "predicateType": "https://model_signing/signature/v1.0",
       "predicate": {
+        "ignore_paths": [
+          "model.sig",
```

Comment:
I know this is a docstring, but would it include the signature file in the ignored paths? right now our verifier implicitly ignores the passed sig file, so I don't think it needs it again.

Reply:
True. I will try to filter it out at some point.

Reply:
Since it's being added early on in _cli.py and we don't have a standardized name `model.sig`, it's  not so easy to remove it later on.